File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file104.f90
program DRB023_sections1_orig_yes
    use omp_lib
    implicit none

    integer :: i
    i = 0

    !$omp parallel sections
    !$omp section
    i=1
    !$omp section
    i=2
    !$omp end parallel sections

    print 100, i
    100 format ("i=",i3)

end program

File: data/gpt-4o/readable_format_HPEC/cpp/file104.cpp
#include <stdio.h>
#include <omp.h>
int main()
{
 int i = 0;
#pragma omp parallel sections
 {
#pragma omp section
 i=1;
#pragma omp section
 i=2;
 }
 printf ("i=%d\n", i);
 return 0;
}
